Assets/Scripts/Photos/PhotoCamera.cs

```diff
@@ -83,13 +117,24 @@ public void Capture()
         RenderTexture.active = myRenderTexture;
 
         byte[] bytes = image.EncodeToPNG();
+        PhotoDTO photo = new PhotoDTO(bytes, defaultPhotoFileFormat);
 
-        // Write photo to photo directory path
-        File.WriteAllBytes(PhotoDirectoryPath + photoCount + ".png", bytes);
-        Debug.Log(PhotoDirectoryPath + photoCount + ".png");
-        photoCount++;
+        Capturable[] capturables = GameObject.FindObjectsOfType<Capturable>();
```

Comment:
If this is actually a legit way to grab these, I'll remove the "Capturable" layer